jardin/instrumentation/base_subscriber.py

```diff
@@ -0,0 +1,8 @@
+
+
+from jardin.instrumentation.event import Event
+
+
+class BaseSubscriber:
```

Comment:
1. Whitespace at the top of the file is weird
2. For a simple callback like this, could we just have the client provide a function that takes an `Event` as a parameter?

Reply:
For 2, I could go either way. But I think using callbacks might make this less future-proof if we want to add more methods to the subscriber